src/ray/raylet/node_manager.cc

```diff
@@ -1133,6 +1136,9 @@ void NodeManager::AssignTask(Task &task) {
       this->cluster_resource_map_[my_client_id].Acquire(spec.GetRequiredResources()));
 
   if (spec.IsActorCreationTask()) {
+    // Check that we are not placing an actor creation task on a node with 0
+    // CPUs.
```

Comment:
do we need to wrap this line? Hard to tell, but looks like < 90chars to me if combined. Ignore if not the case.

Reply:
Fixed.